infra/fendermint/scripts/subnet.toml

```diff
@@ -122,6 +123,10 @@ script.pre = "mkdir -p ${BASE_DIR}/${NODE_NAME}/${KEYS_SUBDIR}; cp ${PRIVATE_KEY
 extend = "fendermint-tool"
 env = { "CMD" = "genesis --genesis-file /data/genesis.json ipc from-parent --subnet-id ${SUBNET_ID} -p ${PARENT_ENDPOINT}  --parent-gateway ${PARENT_GATEWAY}  --parent-registry ${PARENT_REGISTRY} --base-fee ${BASE_FEE} --power-scale ${POWER_SCALE}" }
 
+[tasks.subnet-genesis-set-eam-permissions]
+extend = "fendermint-tool"
+env = { "CMD" = "genesis --genesis-file /data/genesis.json set-eam-permissions --mode unrestricted" }
```

Comment:
@raulk Should we set to allowlist or unrestricted by default?

Reply:
Isn't this exactly what `genesis --genesis-file /data/genesis.json ipc from-parent ` [did by default](https://github.com/consensus-shipyard/ipc/blob/ae750fe9778d2b6dda10768c3b6966ba3c648238/fendermint/app/src/cmd/genesis.rs#L327) ?

Reply:
That's right. So what does this Linear issue ask for? Which addresses should be allowed to deploy contracts in this subnet? @aakoshh @raulk

Reply:
from @raulk in sync: pass an env variable to `cargo make child-validator`

Reply:
from @raulk in sync: pass permission mode as an env variable to `cargo make child-validator`